README.md

```diff
@@ -17,21 +21,175 @@ python --version
 pip --version
 ```
 
-From the root directory run the following command to install the dependencies: `pip install -r requirements.txt`
+From the root directory run the following command to install the
+dependencies: `pip install -r requirements.txt`
 
 You can run the app using this command: `python -m uvicorn src.api.index:app --reload`
 
-Once running you can navigate to `http://127.0.0.1:8000/docs` to view the interactive API documentation.
+Once running you can navigate to `http://127.0.0.1:8000/docs` to view the
+interactive API documentation.
 
+## Setup for local code development
+
+There are some steps that need to be done prior to being able to
+properly run and develop the code in this repository.
+
+The followign is a list of steps that have to happen prior to starting to
```

Comment:
Just noticed a small typo here for `following`.

Reply:
Great! Thanks for catching that. I just fixed it in the latest set of updates.